moduleroot/.gitlab-ci.yml.erb

```diff
@@ -50,7 +50,11 @@ before_script:
 <%       else -%>
   stage: syntax
 <%       end -%>
+<%       if options['docker_image'] -%>
+  image: <%= options['docker_image'] %>
+<%       else -%>
```

Comment:
This seems a little strange since it's happening instead the loop for adding ruby versions but you're overriding the docker_image which is the thing that provides the ruby version.  

Are you using docker images that also provide the needed ruby versions otherwise it seems like this would be confusing?